### PR TITLE
SpaceService: DbContext

### PR DESCRIPTION
**Pull request by mekk1t**
-- installed packages ".EntityFrameworkCore.Design" and ".SqlServer";
-- ran "scaffold" command for the Spaces, MapFiles, Offices and SpaceTypes tables;
-- created SpaceService migration via "dotnet ef migration add SpaceService";
-- added Database models for Maps, Offices, Spaces and SpaceTypes.

### CODE REVIEW

File: OfficeMapApi/SpaceService/SpaceService/Database/SpaceServiceDbContext.cs [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+                    .HasDefaultValueSql("(newid())");
+
+                entity.Property(e => e.Name)
+                    .IsRequired()
+                    .HasMaxLength(50)
+                    .IsUnicode(false);
+            });
+
+            modelBuilder.Entity<DbOffice>(entity =>
+            {
```
Comment (aatroshcenko) [inline OfficeMapApi/SpaceService/SpaceService/Database/SpaceServiceDbContext.cs:60]: Добавь ToTable("Offices", "dbo")

File: OfficeMapApi/SpaceService/SpaceService/Database/SpaceServiceDbContext.cs [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+            if (!optionsBuilder.IsConfigured)
+            {
+                optionsBuilder.UseSqlServer("Server=localhost;Database=DOM;Trusted_Connection=True;");
+            }
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<DbMap>(entity =>
+            {
```
Comment (aatroshcenko) [inline OfficeMapApi/SpaceService/SpaceService/Database/SpaceServiceDbContext.cs:34]: Добавь ToTable("Maps", "dbo")

File: OfficeMapApi/SpaceService/SpaceService/Database/SpaceServiceDbContext.cs [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+                    .HasMaxLength(50)
+                    .IsUnicode(false);
+
+                entity.Property(e => e.Street)
+                    .HasMaxLength(50)
+                    .IsUnicode(false);
+            });
+
+            modelBuilder.Entity<DbSpaceType>(entity =>
+            {
```
Comment (aatroshcenko) [inline OfficeMapApi/SpaceService/SpaceService/Database/SpaceServiceDbContext.cs:92]: Добавь ToTable("SpaceTypes", "dbo")

File: OfficeMapApi/SpaceService/SpaceService/Database/SpaceServiceDbContext.cs [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+                    .HasMaxLength(255)
+                    .IsUnicode(false);
+
+                entity.Property(e => e.SpaceTypeGuid)
+                    .HasColumnName("SpaceTypeGUID")
+                    .HasDefaultValueSql("(newid())");
+            });
+
+            modelBuilder.Entity<DbSpace>(entity =>
+            {
```
Comment (aatroshcenko) [inline OfficeMapApi/SpaceService/SpaceService/Database/SpaceServiceDbContext.cs:111]: Добавь ToTable("Spaces", "dbo")

**Comment by mekk1t** [comment]
These changes are implemented in the commit "GET/spacetypes" in feature/space-service branch.

**Pull request closed without merging**